staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -148,6 +148,14 @@ var (
 		},
 		[]string{"group", "version", "kind"},
 	)
+	WatchEventsSizes = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "apiserver_watch_events_sizes",
+			Help:    "Response size distribution in bytes",
+			Buckets: prometheus.ExponentialBuckets(1000, 2.0, 7), // 1K, 2K, 4K, 8K, ..., 128K.
```

Comment:
nit: s/1000/1024/ maybe?

[Also, I don't think 128K bucket will be there]

Reply:
Yes, the comment either needs to end at 64k or the bucket count needs to be 8..

Reply:
Very good point. Thanks for catching this. I changed to 1024 base and 8 buckets.